handler/core/src/test/java/io/knotx/fragments/handler/action/CircuitBreakerDoActions.java

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
It is duplicate, please check `io.knotx.fragments.handler.action.cb.CircuitBreakerDoActions`.